Feat/add registry work item language and links to templates (#17)
decentralized-identity/org: work-item-lifecycle.md

## Work Item 

A work item can be considered a DIF work item once the status of the item reaches one
of the stages mentioned in the Deliverable Development Process (above). Each DIF
work item must be concluded and developed under the
https://github.com/decentralized-identity organization's repository to be
considered part of the Deliverable Development Process, making it a DIF work
item. 

Types of Work Item:

### Specification

The most common output of working groups at DIF are specifications, which
unambiguously define and specify distinct protocols, data models, and/or

> functional software modules. 

Author: great catch!

profile is adequately descriptive and non-normative to constitute an IPR-safe
donation before it goes to the Technical Steering Committee for DIF
ratification.

### Registry

In some cases, an ongoing work item can created that collects inputs from
outside DIF, such as a directory or registry of links to downstream projects or
governance authorities. To minimize reputational risk or market confusion that
could occur from such open-ended registries bearing an implicit DIF seal of

> approval, registry work items should be approved by working group chairs (and at

Reviewer: ```suggestion
approval, registry work items must define the process for registering items,
this process should be approved by working group chairs (and at
```

discretion of same, the DIF steering committee) before being opened up to
outside PRs.  Once approved, editors (i.e. code owners) of a given registry are

> encouraged to merge PRs and grow their registry indefinitely within guidance at
> time of approval, but to reach out to working group chairs in case of

Reviewer: ```suggestion
encouraged to merge PRs and grow their registry indefinitely within the
guidelines of the approved process, while relying on working group chairs in case of
```
